pages/agent/v3/monitoring.md

```diff
@@ -0,0 +1,49 @@
+# Monitoring and observing the Buildkite Agent
+
+By default, the Buildkite Agent is only observable either through Buildkite or
+ through log output on the host:
+
+- Job logs, which relate to the jobs the agent runs, are uploaded to Buildkite,
+ and are shown for each step in a build.
+- Agent logs, which relate to how the agent itself is running, are not uploaded
+ or saved (except where this is done by another process such as systemd or
+ launchd).
+
+## Health checking and status page
+
+The agent can optionally run a HTTP service that describes the state of the
+agent. The service is suitable for both automated health-checking and human
+inspection.
+
+You can enable the service with the `--health-check-addr` flag or
+`$BUILDKITE_AGENT_HEALTH_CHECK_ADDR` environment variable:
+
+```shell
+buildkite-agent start --health-check-addr=:3901
+```
+
+For security reasons, we recommend that you do **not** expose the service
```

Comment:
Match the emphasis you use below with italics rather than bold here:

```suggestion
For security reasons, we recommend that you do _not_ expose the service
```